tests/MonoGame.Extended.Tests/Content/ContentReaderExtensionsTests.cs

```diff
@@ -22,6 +22,7 @@ public void ContentReaderExtensions_ShortenRelativePath(string input, string exp
             Assert.True(ContentReaderExtensions.ShortenRelativePath(input) == expectedoutput);
         }
 
+#if !KNI
```

Comment:
The ContentReaderExtensions_GetRelativeAssetName test is testing an internal method of MonoGame.
This fails on KNI & FNA.

Reply:
This test should be removed.  I added it initially to verify the issue that was related to it, but we shouldn't be testing internal MonoGame methods.  If they have an issue, it should be tested by them and we'll report on it.

Feel free to remove this test.

Reply:
Done.